src/imitation/scripts/config/parallel.py

```diff
@@ -7,10 +7,8 @@
 Adding custom named configs is necessary because the CLI interface can't add
 search spaces to the config like `"seed": tune.choice([0, 1, 2, 3])`.
 
-For tuning hyperparameters of an algorithm on a given environment, override
-the `base_named_configs` argument with the named config of the environment.
-Ex: python -m imitation.scripts.parallel with example_gail \
-    'base_named_configs=["logging.wandb_logging", "seals_half_cheetah"]'
+For tuning hyperparameters of an algorithm on a given environment,
+check out the benchmarking/tuning.py script.
```

Comment:
Why did you remove the example here?

Reply:
The example was for tuning the HPs using the parallel script which we have now moved to the tuning script along with all the named configs for tuning HPs.